deployments/values/supported-matrix.yaml

```diff
@@ -29,10 +29,10 @@ matrix:
       version: "5"
       pattern: *development_pattern
     adw:
-      version: "4.4"
+      version: "4.5"
       pattern: *development_almost_semver_pattern
     adminApp:
-      version: "8.4"
+      version: "8.5"
```

Comment:
Those are not scheduled for release in fornax

Reply:
I've bumped them thinking that we want them in the acs-deployment pre-release values file, frontends never really get any -A to be released